assets/queries/cloudFormation/filter_and_alarm_exist_VPC_changes/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Filter_And_Alarm_Exist_For_VPC_Changes",
+  "queryName": "Filter And Alarm Not Exists For VPC Changes",
```

Comment:
Change **directory name** and **id** to: `vpc_changes_without_alarms`
Change **query name** to: `VPC Changes Without Alarms`